Tests/Unit/NetworkServer/LoRaDeviceFactoryTest.cs

```diff
@@ -13,8 +14,9 @@ namespace LoRaWan.Tests.Unit.NetworkServer
     using Moq.Protected;
     using Xunit;
 
-    public class LoRaDeviceFactoryTest
+    public sealed class LoRaDeviceFactoryTest : IDisposable
     {
+        private readonly Meter meter = new Meter(MetricRegistry.Namespace, MetricRegistry.Version);
```

Comment:
[nit] Depending on the metric, we might get interference from other tests if we use the metric with the same registry as in other tests. Typically, therefore we use `TestMeter.Instance` for tests since we do not listen on these metrics anywhere.